### ISSUE TITLE
Unable to do add to cart in Quick View page

### ISSUE THREAD

**Issue by jaishivnani** [haris989/MyAwesomeCart#28]
![image](https://user-images.githubusercontent.com/30934818/97082123-f0d42200-1624-11eb-8267-fc45a8f0c549.png)

**Comment by Huzaifa785** [comment]
Yeah, you are right, and also the cart is not updating on every page. If you get the solution, please explain to me too.

**Comment by jaishivnani** [comment]
i Solved it  by adding same javascript but still name is not updating in popover of quick view page and console.log(cart) is showing the following values 
1.	pr: (3) [3, "", 39000] (Name is coming as blank while adding to cart in ProdView)
2.	__proto__: Object

**Comment by abhi-shek-2** [comment]
Hi jaishivnani can you give the JS code for the productview page???

**Comment by jaishivnani** [comment]
Hello Abhishek,

JS code is there on my github for product view page.

giving it again
{% extends 'shop/basic.html' %}
{% block title%} {{product.product_name}} - My Awesome Cart {% endblock %}
{% block body %}
<div class="container my-4">
<div class="row">
<div class="col-md-4">
<div class="row">
<img src="/media/{{product.image}}" width="360px" height="400px">
</div>
<div class="row">
<button class="btn btn-primary mx-3 my-1">Buy Now</button>
<button class="btn btn-primary mx-3 my-1">Add To Cart</button>
</div>
</div>
<div class="col-md-8">
<h5> {{product.product_name}}</h5>
<p><b>Rs.{{product.price}}</b></p>
<p>{{product.desc}}</p>
</div>
</div>
</div>
{% endblock %}
{% block js %}
<script>
console.log("working")
if (localStorage.getItem('cart') == null) {
// < !--creating object array-- >
var cart = {};
}
else {
// < !--Converting string to object array using JSON.parse-- >
cart = JSON.parse(localStorage.getItem('cart'));
document.getElementById('cart').innerHTML = Object.keys(cart).length;
}
$('.cart').click(function () {
console.log('clicked');
var idstr = this.id.toString();
console.log(idstr);
if (cart[idstr] != undefined) {
cart[idstr] = cart[idstr] + 1;
}
else {
cart[idstr] = 1;
}
console.log(cart);
localStorage.setItem('cart', JSON.stringify(cart));
document.getElementById('cart').innerHTML = Object.keys(cart).length;
});
$('#popcart').popover();
document.getElementById("popcart").setAttribute('data-content', '<h5>Cart
for your items in my shopping cart</h5>');
</script>
{% endblock js %}

**Comment by abhi-shek-2** [comment]
mine is also same but idk why it's not working
